### DISCUSSION TITLE
Size limit on album art ?

### DISCUSSION THREAD

**Discussion by minsonj2016** [gerbera/gerbera#3119] [Q&A]
I installed 2.1.0 on PI4b/bookworm .
After import some of the artwork shows up at the 'album' level and some doesnt (see picture) . The artwork looks correct for the individual flac files . Each flac file has the album art embedded in it . I discovered that the size of the 'good' art work is 58,177 and the size of the 'bad' artwork is 86,135 . Am I running up against some file/db limit ?
 I queried the database structures and can find no obvious reference to the artwork in any table .

![Screenshot_20240606_091231_Hi-Fi Cast](https://github.com/gerbera/gerbera/assets/36899484/01a5171b-edfa-4ce7-b532-e32656e8ffef)

**Comment by KarlStraussberger** [comment]
There is no limit in the database because we do not store artwork. It is loaded at runtime from the file. Sometime the player is the problem sometimes the file itself. Regardless of the "standard" file format stuff, sometimes the files are corrupt or store the data in alternate places that the tagging library does not know of. To get that sorted out, look at the server logs and double check in the web ui (it tells you about embedded resources).

**Reply by minsonj2016** [reply]
OK thanks for the quick reply .  If the image shown for  the 'album'  is taken from one of the tracks which track is used ?

**Reply by KarlStraussberger** [reply]
Yes, it is. The first file scanned. Normally.

**Reply by minsonj2016** [reply]
I'm not sure whats going on here . Albums that show no artwork when doing a recursive import sometimes show the artwork if only that album is imported .
Seems to be no consistency in the behavior .

**Reply by KarlStraussberger** [reply]
That's an item on my tasklist. Not sure why but it's somehow depending on "history" whether it works or not.

**Reply by KarlStraussberger** [reply]
Have you already tried out 2.2. Also there have been recent changes on head, which means you may have to subscribe to the "-latest" release channel.
@minsonj2016
